test/publishToNpm.test.js

```diff
@@ -125,7 +125,7 @@ tap.test('Should stop action if package info retrieval fails', async t => {
   const { publishToNpmProxy, runSpawnStub } = setup()
 
   runSpawnStub
-    .withArgs('npm', ['view', 'fakeTestPkg@v5.1.3'])
+    .withArgs('npm', ['view', '--json'])
```

Comment:
is this test still accurate? shall we stop the action if package retrieval fails because of package not existing?

Reply:
isn't clear from the diff view, but the tests check for continue on a 404 from `npm view --json` and stopping for any other error